README.md

```diff
@@ -114,18 +114,24 @@ Note: length checking is performed after replacement
 #### Issue description
 
 - Min length: 1 character
-- Max length: 65536 characters
+- Max length: 65536 codepoints
 
 *The first verified by creating an issue with a very long description and confirmed with an error message.*
 
 
 #### PR body 
 
 - Min length: 0 characters
-- Max length: 262144 characters
+- Max length: 65536 codepoints
 
 *This was verified by making PATCH requests to `/repos/{owner}/{repo}/pulls/{pull_number}` with `"body"` in body. This was also verified by adding longer body (262,145 characters) to a PR body in UI and confirmed with the error: `There was an error posting your comment: Body is too long`.*
 
+### Issue comments
+
+- Min length: 1 character
+- Max length: 262144 bytes (65536-262144 characters depending on UTF8-encoded size)
```

Comment:
Did you forget to change this line as well per https://github.com/dead-claudia/github-limits/pull/13#discussion_r2509402890?